jck/runtime.api/playlist.xml

```diff
@@ -379,13 +379,13 @@
 		<testCaseName>jck-runtime-api-java_applet</testCaseName>
 		<disables>
 			<disable>
-				<comment>Disabled on aix for backlog/issues/486. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
-				<platform>ppc64_aix</platform>
+				<comment>Disabled on aix for backlog/issues/486, Problem Report 147661 for Mac. Awaiting some automation improvements, in the interim these targets will be run manually</comment>
+				<platform>x86-64_mac|pppc64_aix</platform>
```

Comment:
You may want to update pppc64_aix with ppc64_aix, or if its running and passing to remove the exclude, that is unlikely to be working as it is at the moment.

Reply:
Correct pppc64_aix -> ppc64_aix. Interestingly, the exclude was working even with `pppc64_aix`! Thanks for spotting!